### PR TITLE
Migrate VCL 

### PR DESCRIPTION
**Pull request by phiden**

### CODE REVIEW

File: sass/modules/_m-button.scss [hunk 1]
```diff
@@ -34,7 +34,7 @@ button,
 
 .usa-button-secondary-exit {
   &::after {
-    background-image: url(/img/icons/exit-icon-primary.png);
+    background-image: url(/dist/img/icons/exit-icon-primary.png);
```
[the remark this answers is not in the document]
Comment (phiden) [reply sass/modules/_m-button.scss]: Yes. I don't know how to go about fixing it; do you have any ideas I can try to work through?

[the remark this answers is not in the document]
Comment (jbalboni) [reply sass/modules/_m-button.scss]: We might need to make sure the images are copied somewhere when building? This is a part of vets-website build that I copied over and don't understand that well. I'll go poke around and see if I can figure out what's happening.

[the remark this answers is not in the document]
Comment (phiden) [reply sass/modules/_m-button.scss]: Ok, cool. Thanks. How do you feel about my merging this in the meantime?

[the remark this answers is not in the document]
Comment (jbalboni) [reply sass/modules/_m-button.scss]: Sounds good to me.

**Pull request merged**